src/SWP/Bundle/ContentBundle/Resources/config/services.yml

```diff
@@ -326,22 +326,24 @@ services:
         arguments:
             - '%env(json:BLACKLISTED_KEYWORDS)%'
 
-    Aws\S3\S3Client:
+    S3Client:
```

Comment:
we are trying to use full FQCN for service names. Please change it from `S3Client` to `SWP\Bundle\ContentBundle\Factory\S3ClientFactory`.